react/README.md

```diff
@@ -568,6 +638,368 @@ When passing a variable to a prop, the curly braces should **not** be padded by
 
 **[⬆ back to top](#table-of-contents)**
 
+## React `key` prop
+
+### Mandatory `key` prop
+
+When rendering an array of React components, specify the `key` prop for each component in the array to aid React's Virtual DOM diffing algorithm (eslint: [`react/jsx-key`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md)):
```

Comment:
Maybe add a warning that react will output a console error is this isn't followed? Just to reinforce this is more than just a best practice but important for performance :)